api/pom.xml

```diff
@@ -45,6 +45,7 @@
         <findbugs.exclude>exclude.xml</findbugs.exclude>
         <findbugs.threshold>Low</findbugs.threshold>
         <copyright-plugin.version>1.49</copyright-plugin.version>
+	<legal.source.folder>${maven.multiModuleProjectDirectory}</legal.source.folder>
```

Comment:
Who sets maven.multiModuleProjectDirectory?

Based on a little googling, it doesn't appear that this is a built in Maven property.
It looks like it's something that every user needs to set when invoking "mvn".
Is that true?  If so, I recommend using this plugin instead:

                <groupId>org.commonjava.maven.plugins</groupId>
                <artifactId>directory-maven-plugin</artifactId>